esp/scm/ws_logaccess.ecm

```diff
@@ -95,6 +90,39 @@ ESPRequest GetLogsRequest
                           //    USR - User
                           //    PRO - Programmer
                           //    ADT - Audit
+                          //If searching by "BySourceInstance", the SearchByValue should contain the instance of interest
+                          //If searching by "BySourceNode", the SearchByValue should contain the node of interest
+                          //If searching by "ByFieldName", the SearchByValue should contain the field value of interest, and the SearchField 
+                          // should be populated.
+    string SearchField;
+};
+
+ESPStruct LogFilterClause
+{
+    ESPenum LogAccessFilterOperator Operator;
+    ESParray<ESPStruct LogFilterOperation, LogFilterOperation> LogFilterOperations;
+};
+
+/*
+* Provides mechanism to query log entries
+*
+* Caller can query by JobId, component, log event type, target audience, source instance, or host by providing the appropriate
+* enumerated value in the LogCategory field, as well as the targeted value in the SearchByValue field.
+* Caller can also query by field name directly by choosing the 'ByFieldName' option and providing the field name, and desired filter value.
+*
+* SearchbyValue is optional if LogCategory == ALL
+*
+* Caller should restrict the query to target a specific time range specified in the Range field.
+* By default, the first 100 log entries encountered are returned. Caller to pagenize using the
+* LogLineStartFrom field (specifies as zero-indexed start index) and the LogLineLimit (specifies the maximum
+* number of log entries to be returned)
+* Caller can specify which log columns should be reported via the Columns field, all available columns returned by default.
+*
+* The report format can be specified via the Format field: JSON|XML|CSV
+*/
+ESPRequest GetLogsRequest
+{
+    ESParray<ESPStruct LogFilterClause, LogFilter> LogFilters;
     ESPStruct TimeRange Range;
     unsigned LogLineLimit(100);
     int64 LogLineStartFrom(0);
```

Comment:
@jeclrsg - Any comments on these changes, compared to your existing implementation?

Reply:
@jeclrsg the intent here is to provide support for compound queries such as WUDI == W2022XYZ AND (logtype == ERR OR logtype == WRN). 
Fair warning this interface isn't final yet, but it's a good time to provide feedback